### PR TITLE
Guice 5

### PR DESCRIPTION
**Pull request by chibenwa**
Final version.

### CODE REVIEW

**Comment by rouazana** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

File: container/cassandra-guice/pom.xml [hunk 1]
```diff
@@ -0,0 +1,412 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mbaechler) [inline container/cassandra-guice/pom.xml]: I think I already said it : dependencies must be in the java 8 profile

**Comment by aduprat** [comment]
:+1:

**Comment by mbaechler** [comment]
:+1:

**Comment by rouazana** [comment]
Maybe we should reindent the code before rebasing?

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by mbaechler** [comment]
:+1:

**Comment by mbaechler** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

**Comment by rouazana** [comment]
:+1:

**Comment by mbaechler** [comment]
reorganized into GUICE branch

**Pull request closed without merging**